tools/usercss-template.css

```diff
@@ -26,7 +26,7 @@
   custom "Custom" <<<EOT
   var(--bg-custom) EOT;
   none "None" <<<EOT
-  none EOT;
+  #202020 EOT;
```

Comment:
Hi @fregante adding color here actually breaks the intended design of this. If you must change something its at the body entry.

https://github.com/StylishThemes/GitHub-Dark/blob/a4c96edabb6c015aeb6c29d52f85a17c04dd496c/github-dark.css#L11625

So instead of `#222222` change it to your `#202020` Which is what none will mean.